boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020 Jefferson Lee
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+    model = "Arduino Nano 33 BLE (Sense)";
+    compatible = "arduino,arduino_nano_33_ble";
+
+    chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &code_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 24 0>;
+			label = "Red LED";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		spi = &spi2;
+	};
+};
+
+&flash0 {
+	/*
+	 * For more information, see:
```

Comment:
@saltyJeff this link does not seem to exist any more, please remove the comment.